core/views.py

```diff
@@ -727,6 +727,41 @@ def add_announcement(request):
             obj.body = form.cleaned_data['body']
             obj.save()
 
+            recievers = []
+            for user in User.objects.all():
+                recievers.append(user.email)
+
+            # send_mail(obj.title, obj.body, 'greekrhoverify@gmail.com', recievers) - can't use this function if we want to use bcc, but keep it in for now
+
+            truemessage = render_to_string('core/announcement_email.html', {
```

Comment:
To do the changes above I think you'll have to add the target from the form to the context here

Reply:
I think this still needs to be added for target to work